.github/workflows/benchmarks.yml

```
@@ -0,0 +1,149 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Workflow for all benchmark-related jobs. It is designed to be called from
+# the main workflow ci.yml. The concurrency of this workflow is controlled by
+# the caller's job.
+
+name: Benchmarks
+
+on:
+  workflow_call:
+    inputs:
+      runner-group:
+        required: true
+        type: string
+      runner-env:
+        required: true
+        type: string
+      gcs-dir:
+        required: true
+        type: string
+      build-dir:
+        required: true
+        type: string
+      build-dir-archive:
+        required: true
+        type: string
+      build-dir-gcs-artifact:
+        required: true
+        type: string
+      tf-binaries-dir:
+        required: true
+        type: string
+      tf-binaries-archive:
+        required: true
+        type: string
+      tf-binaries-gcs-artifact:
+        required: true
+        type: string
+
+jobs:
+  build_benchmark_suites:
```

I think now that the whole workflow is called "benchmarks" we can drop that from the job names

```suggestion
  build_suites:
```